### DISCUSSION TITLE
Disqus comment system doesn't load automatically

### DISCUSSION THREAD

**Discussion by carolhuang1990** [squidfunk/mkdocs-material#3068] [General]
When I first time visit every web page of my website, the Disqus comment system doesn't load.
I can only see the title "Comments"...

I need to refresh the page to see the Disqus comment, and this is a bad experience for readers, coz they don't know there is a comment system until they refresh the page.

I don't know what is the problem.
Can anyone help?

**Comment by squidfunk** [comment]
Please provide more information like your `mkdocs.yml`, and everything that might be relevant. Are you using instant loading? If so, #2487 might be related.

**Reply by carolhuang1990** [reply]
yes, that is a problem of instant loading..
If I disable instant loading , the comment system displays immediately. 
thanks!

**Reply by squidfunk** [reply]
The issue has been fixed in e71b129433ccdb2ae15b8d9c7b9995ad6a6f04b3.

**Reply by squidfunk** [reply]
Released as part of 7.3.1.
